src/schema_formats/XML/GMP.xml.in

```diff
@@ -22074,6 +22091,37 @@ END:VCALENDAR
             <pattern><t>boolean</t></pattern>
           </ele>
         </ele>
+        <ele>
+          <name>krb5_credential</name>
+          <pattern>
+            <attrib>
+              <name>id</name>
+              <comment>
+                A UUID if there is a credential, otherwise the empty string
+              </comment>
+              <type>uuid_or_empty</type>
+              <required>1</required>
+            </attrib>
+            <e>name</e>
+            <o><e>permissions</e></o>
+            <e>trash</e>
+          </pattern>
+          <ele>
+            <name>name</name>
+            <summary>The name of the Kerberos 5 LSC credential</summary>
+            <pattern><t>name</t></pattern>
+          </ele>
+          <ele>
+            <name>permissions</name>
+            <summary>Permissions the user has on the task</summary>
```

Comment:
```suggestion
            <summary>Permissions the user has on the credential</summary>
```

Reply:
I've fixed this and several similar instances incorrectly referring to a task in [0f82c2e](https://github.com/greenbone/gvmd/pull/2332/commits/0f82c2e93b6495ee404acd0ac65ca85e8b9371e8)